docs/reference/query-languages/esql/esql-lookup-join.md

```diff
@@ -1,7 +1,7 @@
 ---
 navigation_title: "Correlate data with LOOKUP JOIN"
 mapped_pages:
- - https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-enrich-data.html
+ - https://www.elastic.co/guide/en/elasticsearch/reference/8.18/_lookup_join.html
```

Comment:
Just 2x checking, is 8.18 what we want here?

Reply:
thanks yes this is metadata on the page if folks want to switch to the 8.18 version :)

Reply:
gets rendered liked this

<img width="205" alt="Screenshot 2025-04-16 at 12 15 05" src="https://github.com/user-attachments/assets/b116f4ff-f63a-42d8-b563-3bb31f86eb64" />

Reply:
🙏 TIL